platform/broadcom/docker-syncd-brcm/Dockerfile.j2

```diff
@@ -22,6 +22,11 @@ debs/{{ deb }}{{' '}}
 ## TODO: add kmod into Depends
 RUN apt-get install -yf kmod
 
+## BCM SAI 3.7 needs protobuf
+RUN apt-get install -yf protobuf-compiler
+RUN apt-get install -yf libprotobuf-dev
+RUN apt-get install -yf pkg-config
```

Comment:
can you update the sai package to add dpendency so that we do not need to explicit install them here?

Reply:
Sure, ok Will update the libsaibcm Makefile to do the protobuf library install as well. Shall upload the updated sai packages.

Reply:
Updated in the new diff

Reply:
I have enabled the INSTALL_PROTOBUF=1 in the saibcm Makefile to install the libprotobuf in the debian package as well.

ifeq ($(INSTALL_PROTOBUF),)
  INSTALL_PROTOBUF=1
endif

Reply:
I mean the debian control file of the sai package.

Reply:
Added the "Depends: libprotobuf-dev" in the Debian control file for Package: libsaibcm. There are some additional changes needed, 
1. Add the libprotobuf-dev package in the sonic-slave-stretch/Dockerfile.j2
2. Use "install_debian_packages" macro in docker-syncd-brcm/Dockerfile.j2